Classes/BITCrashManager.m

```diff
@@ -1053,7 +1054,7 @@ - (void)triggerDelayedProcessing {
  */
 - (void)invokeDelayedProcessing {
 #if !defined (HOCKEYSDK_CONFIGURATION_ReleaseCrashOnlyExtensions)
-  if ([BITHockeyHelper applicationState] != BITApplicationStateActive) {
+  if (!bit_isRunningInAppExtension() && [BITHockeyHelper applicationState] != BITApplicationStateActive) {
```

Comment:
good catch!

Reply:
applicationState returns BITApplicationStateUnknown if running in extension and thus returning from this immediately and preventing from processing crashes